### DISCUSSION TITLE
@types/simple-icons: Export interface SimpleIcon

### DISCUSSION THREAD

**Discussion by dhruvkb** [DefinitelyTyped/DefinitelyTyped#54724] [Issues with a @types package]
The package for `@types/simple-icons` does not export the interface `SimpleIcon` which means that increasingly we need to do `typeof <icon_name>` or define our own type which is a subset of `SimpleIcon`.

**Exhibit:**
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/484583cd1e4100d364097f2078d916e758a9f862/types/simple-icons/index.d.ts#L7

Exporting this type would make it much easier for users of Simple Icons to be able to reference the icon type in their code.

I'm more than happy to submit a PR if there is maintainer bandwidth to review and merge it, so that it's available in the next release.

@ericcornelissen and @peterblazejewicz do you agree with this idea?

**Comment by ericcornelissen** [comment]
I agree that it would make sense to expose the `SimpleIcon` interface, go ahead and open a PR @dhruvkb 🙂
